app/views/application/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1><%= t('hello') %></h1>
\ No newline at end of file
+<section class="front-section row">
+	<div class="container">
```

Comment:
Yip I see, maybe I've been nitpicky in this case. The true problem is outside of your story. The main layout loads the whole site content in an `container-fluid`, so that you need to use `row` to get rid of that padding. It is not wrong, but for me it is not the `bootstrap` way of doing things. But anyway i will approve this, it is just of matter of taste and the problem of the one who made the main layout.